crates/server/src/admin/service.rs

```diff
@@ -121,7 +121,7 @@ pub(crate) fn setup(
         .route("/refresh-jwt-token", post(refresh_jwt_token_handler))
         .route("/generate-jwt-token", post(generate_jwt_token_handler))
         .layer(AuthSignatureLayer::new(store))
-        .layer(Extension(Arc::clone(&shared_state)));
+        .layer(Extension(shared_state.clone()));
```

Comment:
I'm not sure why this change has been made? Is the `shared_state` no longer an `Arc`? It _should_ be wrapped in an `Arc` in order to be safe (and I'm pretty sure Axum requires it) so why the change? Note, using `.clone()` on reference-counted types is a code smell.

Reply:
It's still an `Arc`, just thought we didn't need the verbose syntax. But, fair point. I'll change it back.

Reply:
Resolved